nvm.sh

```diff
@@ -2317,6 +2381,53 @@ nvm_npmrc_bad_news_bears() {
   return 1
 }
 
+# args: path to check and transform if needed
+nvm_ensure_posix_path() {
+  [ -z "${1-}" ] && nvm_err "Need the location as first parameter" && return 1
```

Comment:
please don’t use && for flow control; this should be an if

Reply:
ok, bad habits ;)